azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeIncomingCallCancelledEvent.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+/**
+ * Incoming call cancelled event.
+ */
+public class CallCompositeIncomingCallCancelledEvent {
+    private final int code;
+    private final int subCode;
+    private final String callId;
+
+    /**
+     * Creates {@link CallCompositeIncomingCallCancelledEvent}.
+     * @param code call end code.
+     * @param subCode call end sub code.
+     * @param callId call id.
+     */
+    public CallCompositeIncomingCallCancelledEvent(final int code,
+                                                   final int subCode,
+                                                   final String callId) {
+        this.code = code;
+        this.subCode = subCode;
+        this.callId = callId;
+    }
+
+    /**
+     * Get code.
+     *
+     * @return code
+     */
+    public int getCode() {
```

Comment:
What are these code and subcode? Error code an subcode? If so, should it be CallCompositeErrorCode instead of int?

Reply:
What use of those by Contoso? Will they understand what to do?

Reply:
These will be 
![image](https://github.com/Azure/communication-ui-library-android/assets/73618019/9a230508-2ede-44bc-adcd-a41cb4a2c81e)